ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCleanup/FantomasSettingsConverterProvider.fs

```diff
@@ -0,0 +1,43 @@
+module JetBrains.ReSharper.Plugins.FSharp.Psi.Features.CodeCleanup.FSharpEditorConfig
+
+open System
+open JetBrains.Application
+open JetBrains.Application.Settings
+open JetBrains.Application.Settings.Calculated.Implementation
+open JetBrains.DataFlow
+open JetBrains.Diagnostics
+open JetBrains.Lifetimes
+open JetBrains.ReSharper.Plugins.FSharp.Services.Formatter
+open JetBrains.ReSharper.Psi.EditorConfig
+
+type FantomasSettingsConverter(fantomasSettingsEntry: SettingsIndexedEntry) =
+  let fSharpPrefix = "fsharp_"
+
+  let run (context: SettingsConvertContext<'a>) _ =
+      let context = context.As<SettingsConvertContext<string>>()
+      Assertion.AssertNotNull(context, "Context expected to be SettingsConvertContext<string>")
+
+      for entry in context.SourceData.FindEntriesByPrefix(fSharpPrefix) do
+        let setting = SettingIndex(fantomasSettingsEntry, entry.Key)
+        context.Target.SetValue(setting, entry.Value)
+
+      true
+
+  interface IEditorConfigConverter with
+    member val Category = "F#" with get, set
+    member x.IsPropertySupported(property) = property.StartsWith(fSharpPrefix, StringComparison.Ordinal)
+    member x.Convert(context) = run context null
+    member x.ConvertAndCheck(context) = run context context
+    member x.ReverseConvert _ = failwith "Not implemented"
```

Comment:
Did you test exporting to EditorConfig with that? Would be bad if we would get errors. If there would be errors, then better do nothing instead of failing.